### ISSUE TITLE
New Translation Setup

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/Door43#166]
This will list all of the resources that need to be setup for a new translation in Door43.

**Comment by jag3773** [comment]
Some items to start
- [ ] tA (see #59)
- [x] OBS (done)
- [x] OBS tN ( done, see #133 )
- [x] OBS tW ( see #134)
- [x] OBS tQ ( see #168)

Anything else?

**Comment by richmahn** [comment]
@phillip-hopper is this still in the process of being done, or waiting on something? tA is the only one left to do, right?

**Comment by phillip-hopper** [comment]
@richmahn Yes, tA is the only one left, and it is on hold.

**Comment by alerque** [comment]
@jag3773 Github protip: if you edit to-do lists into the original post at the top of an issue instead of being in a subsequent comment Github will track it as a progress bar for the issue.

**Comment by jag3773** [comment]
Yep, am aware @alerque ;)

**Comment by phillip-hopper** [comment]
@jag3773: Can this be closed now?
